src/app/features/increase-fee-drawer/components/increase-btc-fee-form.tsx

```diff
@@ -0,0 +1,84 @@
+import { toast } from 'react-hot-toast';
+import { useNavigate } from 'react-router-dom';
+
+import { Stack } from '@stacks/ui';
+import BigNumber from 'bignumber.js';
+import { Formik } from 'formik';
+import * as yup from 'yup';
+
+import { BitcoinTransaction } from '@shared/models/transactions/bitcoin-transaction.model';
+import { RouteUrls } from '@shared/route-urls';
+
+import { useBtcAssetBalance } from '@app/common/hooks/balance/btc/use-btc-balance';
+import { satToBtc } from '@app/common/money/unit-conversion';
+import { TextInputField } from '@app/components/text-input-field';
+import { Caption } from '@app/components/typography';
+import { BitcoinTransactionItem } from '@app/features/activity-list/components/transaction-list/bitcoin-transaction/bitcoin-transaction-item';
+import { useCurrentAccountNativeSegwitIndexZeroSigner } from '@app/store/accounts/blockchain/bitcoin/native-segwit-account.hooks';
+
+import { useBtcIncreaseFee } from '../hooks/use-btc-increase-fee';
+import { IncreaseFeeActions } from './increase-fee-actions';
+
+interface IncreaseBtcFeeFormProps {
+  btcTx: BitcoinTransaction;
+}
+
+const feeInputLabel = 'sats/vB';
+
```

Comment:
```suggestion
const feeInputLabel = 'sats/vB';

interface IncreaseBtcFeeFormProps {
  btcTx: BitcoinTransaction;
}
```

style nit of having props directly above component